patterns/ecs-v1/rails

```diff
@@ -1,13 +1,13 @@
 RUUID \h{32}
 # rails controller with action
-RCONTROLLER (?<controller>[^#]+)#(?<action>\w+)
+RCONTROLLER (?<[rails][controller][class]>[^#]+)#(?<[rails][controller][action]>\w+)
 
 # this will often be the only line:
-RAILS3HEAD (?m)Started %{WORD:verb} "%{URIPATHPARAM:request}" for %{IPORHOST:clientip} at (?<timestamp>%{YEAR}-%{MONTHNUM}-%{MONTHDAY} %{HOUR}:%{MINUTE}:%{SECOND} %{ISO8601_TIMEZONE})
+RAILS3HEAD (?m)Started %{WORD:[http][request][method]} "%{URIPATHPARAM:[url][original]}" for %{IPORHOST:[client][address]} at (?<timestamp>%{YEAR}-%{MONTHNUM}-%{MONTHDAY} %{HOUR}:%{MINUTE}:%{SECOND} %{ISO8601_TIMEZONE})
```

Comment:
`client.address` might be the `source.address` in some cases, 
for (often) proxied requests `client.address` made more sense

Reply:
In ECS we want to establish the baseline of always having source/destination populated. Then if the client/server pair makes semantic sense as well, it can be populate too, in addition (not instead of) source/destination.

In the case of web access logs, it's frequent that the destination or server side isn't populated, though. That's fine.

Reply:
OK, thanks. That's good to know - going to change this one to `source.address` than ...